src/components/ReportActionItem/ReportPreview.js

```diff
@@ -115,23 +118,34 @@ function ReportPreview(props) {
                     accessibilityRole="button"
                     accessibilityLabel={props.translate('iou.viewDetails')}
                 >
-                    <View style={[styles.flexShrink1]}>
-                        {props.iouReport.hasOutstandingIOU ? (
-                            <Text style={[styles.chatItemMessage, styles.cursorPointer, styles.colorMuted]}>
-                                {lodashGet(message, 'html', props.translate('iou.payerOwesAmount', {payer: managerName, amount: reportAmount}))}
-                            </Text>
-                        ) : (
-                            <View style={[styles.flexRow]}>
-                                <Text style={[styles.chatItemMessage, styles.cursorPointer, styles.colorMuted]}>
-                                    {lodashGet(message, 'html', props.translate('iou.payerSettled', {amount: reportAmount}))}
-                                </Text>
+                    <View style={styles.iouPreviewBox}>
+                        <View style={styles.flexRow}>
+                            <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter]}>
+                                {props.iouReport.hasOutstandingIOU ? (
+                                    <Text style={[styles.textLabelSupporting, styles.lh16]}>{props.translate('iou.payerOwes', {payer: managerName})}</Text>
+                                ) : (
+                                    <Text style={[styles.textLabelSupporting, styles.lh16]}>{props.translate('iou.payerPaid', {payer: managerName})}</Text>
```

Comment:
suggested change
instead of 2 <Text> render one like
`<Text style={[styles.textLabelSupporting, styles.lh16]}>{props.translate(props.iouReport.hasOutstandingIOU ? 'iou.payerPaid' :'iou.payerPaid' , {payer: managerName})}</Text>`